src/cryptenroll/cryptenroll.h

```diff
@@ -9,6 +9,7 @@ typedef enum EnrollType {
         ENROLL_PKCS11,
         ENROLL_FIDO2,
         ENROLL_TPM2,
+        ENROLL_KEYRING,
```

Comment:
hmm, so far we only expose our own slot types here, why would we expose the cryptsetup internal one here too?

(what does "internal" mean btw? and which "keyring" btw? the kernel keyring? and why would a reference to the kernel keyring end up in the slot info in the json superblock? not following?)